pkg/cmd/cli/describe/describer.go

```diff
@@ -418,6 +420,36 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 	})
 }
 
+// OAuthAccessTokenDescriber generates information about an OAuth Acess Token (OAuth)
+type OAuthAccessTokenDescriber struct {
+	client.Interface
+}
+////--
+func (d *OAuthAccessTokenDescriber) Describe(namespace, name string) (string, error) {
+	c := d.OAuthAccessTokens()
+	oAuthAccessToken, err := c.Get(name)
+	if err != nil {
+		return "", err
+	}
+
+	return tabbedString(func(out *tabwriter.Writer) error {
+		formatMeta(out, oAuthAccessToken.ObjectMeta)
+		formatString(out, "Client Name", oAuthAccessToken.ClientName)
+		formatString(out, "Expires In", oAuthAccessToken.ExpiresIn)
```

Comment:
Preprocess this value so that its a duration from now or a datetime when its done.  Maybe either depending on distance?  "5 minutes" or "2 days", but "october 6, 2016". 

@fabianofranz do we have a time interpretting helper for this?